gateway-ha/pom.xml

```diff
@@ -140,7 +128,7 @@
         <dependency>
             <groupId>mysql</groupId>
             <artifactId>mysql-connector-java</artifactId>
-            <version>8.0.17</version>
+            <version>${mysqlconnector.version}</version>
```

Comment:
nit: dep.mysql.version

Reply:
we will clean this up separately later .. this PR is minimal